### PR TITLE
Feature/pdct 1026 fix summariesgeographygeo endpoint

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

- Return 404 if no matching geography found
- Remove unused placeholder variable from unpacking that was breaking the API

## Type of change

Please select the option(s) below that are most relevant:

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/api/api_v1/routers/summaries.py [hunk 1]
```diff
@@ -22,24 +24,31 @@
...
+def search_by_geography(request: Request, geography_string: str, db=Depends(get_db)):
     """Searches the documents filtering by geography and grouping by category."""
 
     geography_slug = None
     if is_country_code(db, geography_string):
         geography_slug = get_country_slug_from_country_code(db, geography_string)
 
     if geography_slug is None:
-        geography_slug = geography_string
+        geography_slug = geography_string.lower()
```
Comment (diversemix) [inline app/api/api_v1/routers/summaries.py:35]: Are you sure we shouldn't be fixing how we give the slug to the front-end in the first place?

**Review by diversemix — APPROVED** [review]
Let's get this fix in while we figure out what has changed to break it

File: app/core/browse.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def browse_rds_families(
     _LOGGER.debug("Starting families query")
     families = [
         to_search_response_family(family, geography, organisation)
-        for (family, geography, _, organisation) in query.all()
```
Comment (katybaulch) [inline app/core/browse.py:107]: This line was why we got a 500 error

Comment (diversemix) [reply app/core/browse.py:107]: Thanks - some confusion on my part here

**Pull request merged**